storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3697,6 +3715,12 @@ static int rocksdb_init_func(void* const p)
     mysql_mutex_unlock(&rdb_sysvars_mutex);
   }
 
+  if (rocksdb_persistent_cache_size > 0) {
```

Comment:
Ah, okay. RocksDB checks to see if the path is empty and fails the setup. That would be fine. It wasn't clear what would happen when the empty string was passed in and if RocksDB would create a default one.